actionpack/lib/action_dispatch/testing/request_encoder.rb

```diff
@@ -0,0 +1,54 @@
+module ActionDispatch
+  class RequestEncoder # :nodoc:
```

Comment:
Ah nope, it doesn't use that namespace. Anyway, I think it's fine to keep it as is then. It's nodoc'ed and under the `testing/` folder.